src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -355,20 +356,20 @@ let private createAnnotationTableAtRange (isDark:bool, tryUseLastOutput:bool, ra
 
         // Is user input signals to try and find+reuse the output from the previous annotationTable do this, otherwise just return empty array
         let! prevTableOutput =
-            if (tryUseLastOutput) then getPrevTableOutput context                
-            else promise {return Array.empty}
+            if (tryUseLastOutput) then tryGetPrevTableOutput context                
+            else promise {return None}
     
         // If try to use last output check if we found some output in "prevTableOutput" by checking if the array is not empty.
-        let useExistingPrevOutput = (tryUseLastOutput) && Array.isEmpty >> not <| prevTableOutput
-
+        let useExistingPrevOutput = (tryUseLastOutput) && prevTableOutput.IsSome
```

Comment:
This can be replaced by `prevTableOutput.IsSome`, as this is only Some if `tryUseLastOutput` is also Some

Reply:
fixed